### DISCUSSION TITLE
How to export melody?

### DISCUSSION THREAD

**Discussion by jevgen** [Studio-kalavati/Swaranotebook-bandish-editor#101] [How to] [answered]
Let's say, I concatenated something rag-ish. How to save it in MP3?

**Comment by shark8me** [comment, marked as answer]
I haven't considered that before. Since the audio is generated note-by-note on the browser, there isn't an wav or mp3 file that's generated. As a hack, you could record audio or do a screen-capture while playing the song (by clicking on the ⏯️ button on the top right of the keyboard)
